aspnetcore/blazor/host-and-deploy/webassembly.md

```diff
@@ -35,32 +35,32 @@ Blazor relies on the host to the serve the appropriate compressed files. When us
 * For IIS `web.config` compression configuration, see the [IIS: Brotli and Gzip compression](#brotli-and-gzip-compression) section. 
 * When hosting on static hosting solutions that don't support statically-compressed file content negotiation, such as GitHub Pages, consider configuring the app to fetch and decode Brotli compressed files:
 
-  * Reference the Brotli decoder from the [google/brotli GitHub repository](https://github.com/google/brotli/) in the app.
+  * Obtain the JavaScript Brotli decoder from the [google/brotli GitHub repository](https://github.com/google/brotli/). As of July, 2020, the decoder file is named `decode.min.js` and found in the repository's `js` folder.
```

Comment:
```suggestion
  * Obtain the JavaScript Brotli decoder from the [google/brotli GitHub repository](https://github.com/google/brotli/). As of July 2020, the decoder file is named `decode.min.js` and found in the repository's `js` folder.
```

I think technically you don't need the comma in month year formats but I might be wrong here. It reads better without the extra comma.

Reply:
Yes, that's a Chicago style, not found actually in the MS manual. We default to Chicago for anything the MS doc cats didn't cover.